api-module/src/main/java/org/devridge/api/domain/sociallogin/context/OAuth2TokenContext.java

```diff
@@ -0,0 +1,43 @@
+package org.devridge.api.domain.sociallogin.context;
+
+import org.devridge.api.domain.sociallogin.dto.request.SocialLoginRequest;
+import org.devridge.api.domain.sociallogin.dto.response.oauth.OAuth2TokenResponse;
+import org.devridge.api.domain.sociallogin.strategy.token.GoogleTokenProvider;
+import org.devridge.api.domain.sociallogin.strategy.token.KakaoTokenProvider;
+import org.devridge.api.domain.sociallogin.strategy.token.NaverTokenProvider;
+import org.devridge.api.domain.sociallogin.strategy.token.OAuth2TokenStrategy;
+import org.devridge.api.exception.sociallogin.SocialLoginException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class OAuth2TokenContext {
+
+    private Map<String, OAuth2TokenStrategy> strategies;
+
+    @Autowired
+    public OAuth2TokenContext(
+            NaverTokenProvider naverTokenProvider,
+            GoogleTokenProvider googleTokenProvider,
+            KakaoTokenProvider kakaoTokenProvider)
```

Comment:
이 부분 코드 스타일 조금만 통일해주시면 될 것 같아요!
```suggestion
            KakaoTokenProvider kakaoTokenProvider
     )
```